src/libraries/Microsoft.Extensions.Configuration.Abstractions/README.md

```diff
@@ -1,47 +1,11 @@
 # Microsoft.Extensions.Configuration.Abstractions
 
-Provides abstractions of key-value pair based configuration. Interfaces defined in this package are implemented by classes in [Microsoft.Extensions.Configuration](https://www.nuget.org/packages/Microsoft.Extensions.Configuration/) and other configuration packages.
-
-Commonly used types:
-
-- [Microsoft.Extensions.Configuration.IConfiguration](https://learn.microsoft.com/dotnet/api/microsoft.extensions.configuration.iconfiguration)
-- [Microsoft.Extensions.Configuration.IConfigurationBuilder](https://learn.microsoft.com/dotnet/api/microsoft.extensions.configuration.iconfigurationbuilder)
-- [Microsoft.Extensions.Configuration.IConfigurationProvider](https://learn.microsoft.com/dotnet/api/microsoft.extensions.configuration.iconfigurationprovider)
-- [Microsoft.Extensions.Configuration.IConfigurationRoot](https://learn.microsoft.com/dotnet/api/microsoft.extensions.configuration.iconfigurationroot)
-- [Microsoft.Extensions.Configuration.IConfigurationSection](https://learn.microsoft.com/dotnet/api/microsoft.extensions.configuration.iconfigurationsection)
+Provides abstractions of key-value pair based configuration.
 
 Documentation can be found at https://learn.microsoft.com/dotnet/core/extensions/configuration
 
-## Example
```

Comment:
>Example [](http://example.com/codeflow?start=3&length=7)

why removing the example from here? I know you have moved it to package.md but I am still seeing a value keeping the samples in the source readme.

Reply:
It's to avoid duplication, and also because library Readme for other areas does not have examples. Do you think it's better to duplicate them? Or maybe add link to package readme in library readme, so the former will be more discoverable when browsing on Github?

Reply:
I don't mind duplicating the samples as they are not big anyway.

Reply:
OK, i've restored samples